docs/using.md

```diff
@@ -5,86 +5,52 @@ overview of different components and services.
 
 <!-- MarkdownTOC -->
 
-- [Managing your OpenShift instance](#managing-your-openshift-instance)
-  - [Starting OpenShift](#starting-openshift)
-  - [Stopping OpenShift](#stopping-openshift)
-  - [Deleting OpenShift](#deleting-openshift)
-  - [Updating OpenShift configuration](#updating-openshift-configuration)
+- [Managing Minishift](#managing-minishift)
+  - [Minishift start](#minishift start)
+  - [Minishift stop](#minishift-stop)
+  - [Minishift delete](#minishift-delete)
 - [Minishift runtime options](#minishift-runtime-options)
   - [Flags](#flags)
   - [Environment variables](#environment-variables)
   - [Persistent configuration](#persistent-configuration)
     - [Setting persistent configuration values](#setting-persistent-configuration-values)
     - [Unsetting persistent configuration values](#unsetting-persistent-configuration-values)
-- [Interacting with OpenShift](#interacting-with-openshift)
-  - [OpenShift client binary \(oc\)](#openshift-client-binary-oc)
-  - [Login](#login)
-  - [Console](#console)
-  - [Services](#services)
 - [HTTP/HTTPS Proxies](#httphttps-proxies)
 - [Mounted host folders](#mounted-host-folders)
   - [Mounting custom shared folders](#mounting-custom-shared-folders)
 - [Networking](#networking)
-- [Persistent volumes](#persistent-volumes)
-- [Private container registries](#private-container-registries)
 
 <!-- /MarkdownTOC -->
 
-<a name="managing-your-openshift-instance"></a>
-## Managing your OpenShift instance
+<a name="managing-minishift"></a>
+## Managing Minishift
 
-This section contains information about basic virtual machine and OpenShift management operations.
+When you use Minishift, you interact with two distinct components, both of which are managed by Minishift:
+- the virtual machine(VM) created by Minishift and
+- the OpenShift instance provisioned by the VM.
```

Comment:
Is it not provisioned "by" the VM. Maybe on or within the VM. Otherwise OpenShift is provisioned by Minishift